tests/ara/test_columns.py

```diff
@@ -19,6 +16,7 @@
     ComponentQuantityColumn(data_source="formula", component_name="Si", normalize=True),
     NthBiggestComponentNameColumn(data_source="formula", n=1),
     NthBiggestComponentQuantityColumn(data_source="formula", n=2),
+    MolecularStructureColumn(data_source="molecule", format=ChemicalDisplayFormat.SMILES)
```

Comment:
Should we have a replicate column here so that we exercise both pathways?

Reply:
There aren't meaningful differences between the two as far as this repo is concerned, so I don't see much point in adding it. I wrote a script which builds a table with both columns, which I will convert to an e2e test following the merge of this PR.